ansible/config_sonic_basedon_testbed.yml

```diff
@@ -238,3 +368,30 @@
         shell: config save -y
         when: save is defined and save|bool == true
     when: deploy is defined and deploy|bool == true
+
+- hosts: [ptf]
```

Comment:
NO familiar with py test.

Can you explain where does these actions happen? Thanks!

Copy all certs on localhost
Copy certs from localhost to SONiC DUT
Copy certs from localhost to PTFDocker

Reply:
1. All certs creation contains delegate_to: localhost this will copy certs to localhost
2. line# 237 onwards  (copy server_key from local to remote) contains copy to sonic DUT
3. starting hosts: ptf contains code block to copy to ptfdocker but this is currently not working.